scripts/s3.py

```diff
@@ -74,13 +84,24 @@ def wrap_vault(vault: Union[VaultV1, VaultV2], samples: ApySamples, aliases: dic
         "type": "v2" if isinstance(vault, VaultV2) else "v1",
         "emergency_shutdown": vault.vault.emergencyShutdown() if hasattr(vault.vault, "emergencyShutdown") else False,
         "updated": int(time()),
+        "migrations_available": migrations_available,
+        "latest_vault_address": latest_vault_address,
     }
 
     if any([isinstance(vault, t) for t in [Backscratcher, YveCRVJar]]):
         object["special"] = True
-    
+
     return object
 
+def getAssetsMetadataDict(v2_vaults: list) -> dict:
+    v2_registry_adapter = Contract('0x240315db938d44bb124ae619f5fd0269a02d1271')
```

Comment:
We should ideally resolve `lens.ychad.eth` using ENS instead of hardcoding the address